web/client/codechecker_client/thrift_call.py

```diff
@@ -58,14 +59,15 @@ def wrapper(self, *args, **kwargs):
                     codechecker_api_shared.ttypes.ErrorCode.UNAUTHORIZED:
                 LOG.error('Unauthorized to access\n %s',
                           str(reqfailure.message))
+                LOG.error('Ask the product admin for '
+                          'additional access rights.')
```

Comment:
Use as long as lines as it is possible.
```suggestion
                LOG.error('Ask the product admin for additional access '
                          'rights.')
```